src/system/system.types.ts

```diff
@@ -0,0 +1,25 @@
+import type { Temporal } from "temporal-polyfill";
+import type { NilDid } from "#/common/nil-did";
+
+/**
+ *
+ * Controller types
+ */
+export type MaintenanceDocument = {
+  _id: NilDid;
```

Comment:
Probably ObjectId is what we want here ... uuid for ids shared between nodes, dids for entity identifiers and objectid for things internal to the node